exceltesting_test.go

```diff
@@ -258,38 +257,31 @@ func Test_exceltesing_DumpCSV(t *testing.T) {
 			name: "dumped",
 			args: args{r: DumpRequest{TargetBookPaths: []string{filepath.Join("testdata", "dump.xlsx")}}},
 			want: filepath.Join("testdata", "want_dump_会社.csv"),
+			got:  filepath.Join("testdata", "csv", "dump_会社.csv"),
 		},
 
 		{
 			name: "dumpedWithEmptyFile",
 			args: args{r: DumpRequest{TargetBookPaths: []string{filepath.Join("testdata", "dumpWithEmptyFile.xlsx")}}},
 			want: filepath.Join("testdata", "want_dumpWithEmptyFile_会社.csv"),
+			got:  filepath.Join("testdata", "csv", "dumpWithEmptyFile_Sheet1.csv"),
 		},
 	}
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			e := &exceltesing{nil}
-			for _, targetPath := range tt.args.r.TargetBookPaths {
-				ef, err := excelize.OpenFile(targetPath)
-				if err != nil {
-					t.Errorf("exceltesing: excelize.OpenFile: %v", err)
-					return
-				}
-				defer ef.Close()
-
-				for _, sheet := range ef.GetSheetList() {
-					outDir := filepath.Join(filepath.Dir(targetPath), "csv")
-					outFileName := fmt.Sprintf("%s_%s.csv", getFileNameWithoutExt(targetPath), sheet)
-					tt.got = filepath.Join(outDir, outFileName)
-				}
-			}
 			e.DumpCSV(t, tt.args.r)
 			b1, err := os.ReadFile(tt.want)
 			if err != nil {
 				t.Errorf("read file: %v", tt.want)
 				return
 			}
+			_, EMPTY := os.Stat(tt.got)
+			if os.IsNotExist(EMPTY) {
```

Comment:
お、むっちゃいい感じに判定できましたね！
ありがとうございます！

変数名は `err` が良いでしょう。os.Stat() の戻り値は `Stat(name string) (FileInfo, error)` であり、`error` が返ってきているので。

```go
	_, err = os.Stat(tt.got)
	if os.IsNotExist(err) {
		fmt.Printf("%v is not found, because it is empty.\n", filepath.Base(tt.got))
		return
	}
```